directives/annotated.py

```diff
@@ -74,7 +79,10 @@ def run(self):
 
     def assert_sanity(self, content):
         annotation_numbers_present = set(map(lambda matching: int(matching[0]), re.findall(u"\d«", content)))
-        highest_present = max(annotation_numbers_present)
+        if len(annotation_numbers_present) == 0:
+            highest_present = 1
```

Comment:
Why is this `if` necessary?

Reply:
It is is not necessary, it is good that you saw this change. I just forgot to remove this line that I added when I was trying something.